desktop/release-notes.md

```diff
@@ -24,12 +24,118 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.12.0
+2022-09-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+### New
+
+- Added the ability to use containerd for pulling and storing images. This is an experimental feature. 
+- Docker Desktop now runs untagged images. Fixes [https://github.com/docker/for-mac/issues/6425]
+- Added search capabilities to Docker Extension's Marketplace. Fixes [docker/roadmap#346](https://github.com/docker/roadmap/issues/346).
+-  Compose V2 is now enabled by default on new installations of Docker Desktop.
```

Comment:
```suggestion
- Compose V2 is now enabled by default on new installations of Docker Desktop.
- Precedence order of environment variables in Compose is more consistent, and clearly [documented](https://docs.docker.com/compose/envvars-precedence/).
```

Sorry for the late addition to the release notes! The updated version of Compose included in this release will include some changes to the implementation of environment variables. 

In the transition from v1 to v2, we received some user feedback that existing env var usage wasn't resulting in expected behavior. Since the evolution of the env vars implementation in v1 was quite organic, the rewrite in Go for v2 resulted in subtle changes. Instead of replicating the exact edge cases from v1, we address this by clearly documenting the precedence order in the latest version of Compose. 

It's debatable whether this is actually "New," but I placed the item here for now since we want users to be aware of this change. If you think it's more appropriate for another section, feel free to move it.

Reply:
Thanks for the addition Han. Wording LGTM but think this should be moved to the 'Bugs and minor changes' section @andrei-z-albu